lib/sectorbuilder/sectorbuilder.go

```diff
@@ -142,17 +144,21 @@ func New(cfg *Config, ds dtypes.MetadataDS) (*SectorBuilder, error) {
 	}
 
 	var lastUsedID uint64
-	b, err := ds.Get(lastSectorIdKey)
-	switch err {
-	case nil:
-		i, err := strconv.ParseInt(string(b), 10, 64)
-		if err != nil {
+	if cfg.OverrideLastID != 0 {
```

Comment:
This feels hacky, I'd prefer writing that to the datastore when constructing sectorbuilder

Reply:
I've changed it apply when nothing in the ds is found (and changed the name).

Reply:
I'd still prefer writing to the datastore in preseal, but fine..